apps/meteor/app/api/server/v1/teams.ts

```diff
@@ -242,8 +242,9 @@ API.v1.addRoute(
 			const canUpdateAny = !!(await hasPermissionAsync(this.userId, 'view-all-team-channels', team.roomId));
 
 			const room = await Team.updateRoom(this.userId, roomId, isDefault, canUpdateAny);
+			const numberOfMembers = await TeamMember.countDocuments({ teamId: team._id });
```

Comment:
Here to get the total number of team-members, instead of querying the database again, you can return the total members from the above `Team.updateRoom` function call. The above function already has that info.